src/server/actions/study.actions.ts

```diff
@@ -189,14 +189,16 @@ export const approveStudyProposalAction = memberAction(async (studyId: string) =
 
         let status: StudyJobStatus = 'CODE-APPROVED'
 
-        if (USING_S3_STORAGE) {
+        // TODO Is code build part of s3? do we wanna skip this on local/ci?
+        //  maybe we want to use SIMULATE_IMAGE_BUILD ?? its unused right now
+        if (DEV_ENV) {
+            status = 'JOB-READY' // if we're not using s3 then containers will never build so just mark it ready
+        } else {
```

Comment:
code build is not really part of s3, it's a codebuild job that reads the code files from the bucket.  I do think we can use SIMULATE_IMAGE_BUILD vs DEV_ENV as you note